xclim/core/calendar.py

```diff
@@ -834,6 +835,58 @@ def construct_offset(mult: int, base: str, start_anchored: bool, anchor: str | N
     )
 
 
+def offset_is_divisor(freqA: str, freqB: str):
```

Comment:
I suggest using descriptive names for freqA and freqB. 
freqA -> divisor
freqB -> freq

Also, the function name could be `is_offset_divisor`, to emulate other `is_condition` functions found elsewhere.